### PR TITLE
user_number not required

### PR DESCRIPTION
**Pull request by sreynen**
This allows Spoke to work with a Twilio messaging service. In a messaging service, the sender number is chosen by Twilio rather than Spoke, so these changes should all make Spoke work in that context.

### CODE REVIEW

**Review by schuyler1d — APPROVED** [review]
:+1: looks good!

File: src/workers/message-sender-234.js [hunk 1]
```diff
@@ -9,8 +9,8 @@ const serviceMap = { nexmo, twilio }
 async function sendMessages() {
   const messages = await r.table('message')
     .getAll('QUEUED', { index: 'send_status' })
-    .filter((doc) => doc('user_number').match('[234]$'))
-    .group('user_number')
+    .filter((doc) => doc('contact_number').match('[234]$'))
```
Comment (shakalee14) [inline src/workers/message-sender-234.js:12]: can you remind me again why we we're matching for 234 and 56 specifically?

File: src/workers/message-sender-789.js [hunk 2]
```diff
@@ -9,8 +9,8 @@ const serviceMap = { nexmo, twilio }
 async function sendMessages() {
   const messages = await r.table('message')
     .getAll('QUEUED', { index: 'send_status' })
-    .filter((doc) => doc('user_number').match('[789]$'))
-    .group('user_number')
+    .filter((doc) => doc('contact_number').match('[789]$'))
```
Comment (shakalee14) [inline src/workers/message-sender-789.js:12]: See above comment about matching for these specific numbers - these arent area codes?

File: src/workers/message-sender-234.js (hunk 1 above)
Comment (sreynen) [reply src/workers/message-sender-234.js:12]: As far as I can tell, the message-sender workers are just broken up into arbitrary groups to spread the load among a few workers, and the last digit in a phone number has relatively even distribution.

File: src/workers/message-sender-789.js (hunk 2 above)
Comment (sreynen) [reply src/workers/message-sender-789.js:12]: No, it's the last digit of the phone number.

File: src/workers/message-sender-234.js (hunk 1 above)
Comment (shakalee14) [reply src/workers/message-sender-234.js:12]: yep, it's just counting up! great, thanks.

**Review by shakalee14 — APPROVED** [review]

**Pull request merged**